internal/backend/rclone/config.go

```diff
@@ -12,13 +12,15 @@ type Config struct {
 	Program     string `option:"program" help:"path to rclone (default: rclone)"`
 	Args        string `option:"args"    help:"arguments for running rclone (default: serve restic --stdio --b2-hard-delete)"`
 	Remote      string
+	Timeout     uint `option:"timeout"     help:"set a timeout limit (in mins) to wait for rclone to establish a connection (default: 1)"`
```

Comment:
Please use `time.Duration`. Then a user can specify `60s` or `1m` which makes clear which unit is used.